main/src/cgeo/geocaching/filters/gui/FilterViewHolderCreator.java

```diff
@@ -168,7 +169,7 @@ public static void setListInfo(final Collection<Geocache> filteredList, final bo
         listInfoIsComplete = isComplete;
     }
 
-    private static IFilterViewHolder<?> createTerrainDifficultyFilterViewHolder() {
+    private static IFilterViewHolder<?> create1to5ItemRangeSelectorViewHolder() {
```

Comment:
Nice reusage idea :-)

Reply:
This was the reason why I did not even look for its own ViewHolder originally - I thought that the UI was associated implicitly by using a `NumberRangeGeocacheFilter`. But adding it was simple enough, just had to know that it's necessary.
So far my first coding experience with the new filter framework was a real pleasure - thanks for putting all this effort into it!